src/p_saveg.cpp

```diff
@@ -512,10 +512,7 @@ void P_SerializeWorld (FArchive &arc)
 				<< si->Index;
 			DBaseDecal::SerializeChain (arc, &si->AttachedDecals);
 			// [BC]
-			arc << si->SavedFlags
-				<< si->SavedTopTexture
-				<< si->SavedMidTexture
-				<< si->SavedBottomTexture;
```

Comment:
Since you just moved these into `side_t::part`, shouldn't this serialization be added to the serialization of that? Also, I'd assume the other newly added variables need to be added to the serialization as well.

Reply:
I might be wrong, but I believe `si->textures[side_t::top] << si->textures[side_t::mid] << si->textures[side_t::bottom]` already serializes these variables. It's a few lines above this.